pkg/kubelet/apis/kubeletconfig/v1beta1/types.go

```diff
@@ -419,6 +436,10 @@ type KubeletConfiguration struct {
 	// Default: 5
 	// +optional
 	ContainerLogMaxFiles *int32 `json:"containerLogMaxFiles,omitempty"`
+	// ManagerMode is a mode in which internal managers (secret, configmap) are running.
+	// Default: "Watching"
+	// +optional
+	ManagerMode KubeletManagerMode `json:"managerMode,omitempty"`
```

Comment:
The name `ManagerMode` seems too generic. Can we make it more specific for ConfigMap & Secrets?

Reply:
Also, is there any value to configure secrets and configmaps differently? I'm guessing not...

Reply:
Changed to ConfigMapAndSecretManagerMode - if you have any better suggestion, I can change one more time.

Regarding configuring differently - I consciously merge those two - I think we don't want to diverge them.
Let me know if you disagree with that.